pages/requiredfields.js

```diff
@@ -1448,7 +1490,8 @@ function RequiredFields(props) {
                     Close
                   </Button>
                   {editMode ? (
-                    <Button variant="primary" onClick={handleSaveChanges}>
```

Comment:
Why did you remove the onClick method? Without this, we wont call the method when the user clicks the button.

Reply:
it works I had it in two places by mistake